org.gnome.Calculator.json

```diff
@@ -1,11 +1,8 @@
 {
     "id": "org.gnome.Calculator",
     "runtime": "org.gnome.Platform",
-    "runtime-version": "46",
+    "runtime-version": "47",
     "sdk": "org.gnome.Sdk",
-    "sdk-extensions": [
-        "org.freedesktop.Sdk.Extension.vala"
-    ],
     "build-options": {
         "prepend-path": "/usr/lib/sdk/vala/bin",
         "prepend-ld-library-path": "/usr/lib/sdk/vala/lib"
```

Comment:
These two options are only needed for the extension, you can drop them if you don't want to use it.